pkg/scheduler/framework/job_updater.go

```diff
@@ -98,6 +99,10 @@ func (ju *jobUpdater) updateJob(index int) {
 	job := ju.jobQueue[index]
 	ssn := ju.ssn
 
+	if !conf.EnabledActionMap["enqueue"] && job.PodGroup.Status.Phase == scheduling.PodGroupPending {
```

Comment:
please add comments here or extract a small func for this special treatment

Reply:
add comments for this block